canonical/ubuntu.com
# Fix iot redirect links

## Done

- Fix incorrect /iot redirects to /internet-of-things 

## QA

- Go to demos
- Go to /iot/networking, /iot/smart-city, /iot/management
- Check that it redirects correctly to `/internet-of-things/networking`

## Issue / Card

Fixes [WD-13133](https://warthogs.atlassian.net/browse/WD-13133)


## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-1311]: https://warthogs.atlassian.net/browse/WD-1311?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[WD-13133]: https://warthogs.atlassian.net/browse/WD-13133?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
